src/test/java/igrad/logic/commands/EditCommandTest.java

```diff
@@ -0,0 +1,179 @@
+package igrad.logic.commands;
```

Comment:
This class also has the same issue of static imports, please retain them (as per the original code-base), it makes the code much cleaner, and readable.